docs/quick-start/write-data.txt

```diff
@@ -0,0 +1,97 @@
+.. _laravel-quick-start-write-data:
+
+=====================
+Write Data to MongoDB
+=====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create the Function to Insert Data
+
+      Update the ``store()`` method ``MovieController.php``, located in
+      the ``app/Http/Controllers`` directory to insert request data
+      into the database as shown in the following code:
```

Comment:
S: I think the current wording is slighly unclear, something like this might work:
```suggestion
      Navigate to the ``MovieController.php`` file located in
      the ``app/Http/Controllers`` directory. Then, update the ``store()`` method to insert request data
      into the database as shown in the following code:
```

Reply:
I can see how "navigate to" might be a step in an IDE in which you need to expand folder contents, but on the command line, there's no need to change directory to open a file for editing, so I chose to stay away from that wording.

Which part was unclear, and do you think this could be a good alternative (omits describing what they're accomplishing and focuses only on the actions they need to take to proceed):

"Replace the store() method in the MovieController.php file, located in the app/Http/Controllers" directory with the following code:"

Reply:
Okay that makes sense. The main issue I had was the "update the `store()` method `MovieController.php`" part of the sentence since it seemed incomplete (but rewording to "Replace the store() method in the MovieController.php file" fixes that issue). I also think there's a missing comma after "directory,"  but otherwise the alternative wording you gave works for me!